client/src/charts/NivoCharts.js

```diff
@@ -40,18 +41,28 @@ const get_scale_bounds = (stacked, raw_data, zoomed) => {
   };
 };
 
-const default_tooltip = (tooltip_items, formatter) => (
+
+const default_tooltip = (tooltip_items, formatter, total) => ( // total indicates percent value tooltip being used
   <div style={{color: window.infobase_color_constants.textColor}}>
-    <table style={{width: '100%', borderCollapse: 'collapse'}}>
+    <table style={{width: '100%', borderCollapse: 'collapse', fontSize: window.innerWidth < breakpoints.minSmallDevice ? '14px' : 'inherit'}}>
       <tbody>
         { tooltip_items.map(
           tooltip_item => ( 
             <tr key = {tooltip_item.id}>
               <td style= {{padding: '3px 5px'}}>
                 <div style={{height: '12px', width: '12px', backgroundColor: tooltip_item.color}} />
               </td>
-              <td style={{padding: '3px 5px'}}> {tooltip_item.id} </td>
-              <td style={{padding: '3px 5px'}} dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
+              { window.innerWidth < breakpoints.minSmallDevice ?
+                  <td>
+                    <div style={{padding: '3px 5px'}}> {tooltip_item.id} </div>
+                    <div style={{padding: '3px 5px'}} dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
+                  </td>
+                  :
+                  <Fragment>
+                    <td style={{padding: '3px 5px'}}> {tooltip_item.id} </td>
+                    <td style={{padding: '3px 5px'}} dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
+                  </Fragment>
+              }
             </tr>
```

Comment:
In practice, the approach of using conditionals on `window.innerWidth` at render time is probably fine in this case, since the popups are fairly transitory and screen resizes are uncommon\*. In general, the problem with this approach is that there's nothing to make this re-render in response to screen size changing.

\* there is one case where you could have a popup active across a screen resize, and it's when you tap to open a popup on a mobile device and then rotate the device while the popup is still open. Probably relevant case to consider here.

Suggestion 1: move the popup styling in to css so you can use a media query for the condition currently on line 47.

Suggestion 2: use the `MediaQuery` react component in the place of the check on line 55, it'll handle re-rendering it's children on screen resize.